web/documentserver-example/nodejs/app.js

```diff
@@ -969,6 +970,43 @@ app.get("/editor", function (req, res) {  // define a handler for editing docume
     }
 });
 
+app.post("/rename", function (req, res) { //define a handler for renaming file
+
+    var newfilename = req.body.newfilename;
+    var dockey = req.body.dockey;
+    var urlModule = require("url");
+
+    var params = {  // create a parameter object with command method and the document key value in it
+        c: "meta",
+        key: dockey,
+        meta: {
+            title: newfilename
+        }
+    };
+
+    var uri = siteUrl + configServer.get('commandUrl');  // get the absolute command url
```

Comment:
You must use `commandRequest`
https://github.com/ONLYOFFICE/document-server-integration/blob/9a52a538cba3c5b622255bfb1bc0ed216c3080ee/web/documentserver-example/nodejs/helpers/documentService.js#L171